dex-load/src/main/resources/reinstallExtension.sh

```diff
@@ -0,0 +1,7 @@
+systemctl stop waves-devnet.service || true
+dpkg -r waves-dex-extension-devnet || true
+dpkg -P waves-dex-extension-devnet || true
```

Comment:
`dpkg -P waves-dex-extension-devnet || true` is sufficient, see https://askubuntu.com/a/128381

Reply:
OK